style.css

```
@@ -0,0 +1,138 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-style: italic;
+}
+
+.container {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+}
+
+.container .slider {
+  position: absolute;
+  inset: 0;
+
+  width: 500%;
+  height: 100%;
+
+  animation: animate 10s linear infinite;
```

Try using this animation effect, will look better and pleasing (ease-in-out)
```suggestion
  animation: animate 10s linear infinite;
  animation: animate 10s ease-in-out infinite;```

---

done!